deposit_contract.sol

```diff
@@ -80,8 +79,10 @@ contract DepositContract is IDepositContract {
         require(deposit_count < MAX_DEPOSIT_COUNT);
 
         // Check deposit amount
+        require(msg.value >= 1 ether);
```

Comment:
I think it would still make sense keeping `MIN_DEPOSIT_AMOUNT = 1 ether` as a const?

Reply:
I don't like the indirection of these constants tbh

Reply:
How about we keep the constants and clear it up a separate PR?

I think we could
- remove the `_LENGTH` constants after #10
- remove `MAX_DEPOSIT_COUNT` and making it part the require statement prior to typecasting 
- and also drop the `MIN_DEPOSIT_AMOUNT`

The only two constants which make sense are `GWEI` (too late to get https://github.com/ethereum/solidity/issues/8716 done) and `DEPOSIT_CONTRACT_TREE_DEPTH`.

Reply:
Yes. That sounds like a good idea. I'd maybe keep `MAX_DEPOSIT_COUNT`, but let's take that when we get there